docs/src/main/sphinx/sql/explain.rst

```diff
@@ -404,7 +406,8 @@ EXPLAIN (TYPE DISTRIBUTED, FORMAT JSON)
 
 .. warning:: The output format is not guaranteed to be backward compatible across Trino versions.
 
-JSON Distributed plan::
+Process the supplied query statement and create a distributed plan in JSON
+format. The distributed splits the plan into stages, and therefore explicitly shows the data exchange between workers::
```

Comment:
The distributed splits the plan -> The distributed plan splits the logical plan

Reply:
Doesn't seem to be addressed

Reply:
done